sda-svc/templates/ingest-secrets.yaml

```diff
@@ -1,18 +1,14 @@
 {{- if or (or (eq "all" .Values.global.deploymentType) (eq "internal" .Values.global.deploymentType) ) (not .Values.global.deploymentType)}}
-{{ if not .Values.global.confFile }}
----
+{{- if not .Values.global.confFile }}
```

Comment:
If you do it this way it will always create  a secret, even when it is not needed.

Reply:
I am not sure if I understand this. But regardless it can be fixed later